art/estimators/classification/pytorch.py

```diff
@@ -123,6 +123,15 @@ def __init__(
             self._reduce_labels = False
 
     @property
+    def get_device(self):
```

Comment:
If we make this a property we can just call it `device`.
```suggestion
    def device(self):
```